services/visual-studio-marketplace/visual-studio-marketplace-version.service.js

```diff
@@ -27,15 +41,22 @@ export default class VisualStudioMarketplaceVersion extends VisualStudioMarketpl
     return renderVersionBadge({ version })
   }
 
-  transform({ json }) {
+  transform({ json }, includePrereleases) {
     const { extension } = this.transformExtension({ json })
-    const version = extension.versions[0].version
+    const preRelease = 'Microsoft.VisualStudio.Code.PreRelease'
```

Comment:
I think the intent here would be a bit clearer if we called this var `preReleaseKey` or `preReleaseTag`

Reply:
I agree. Fixed.